assets/radar-rules.js

```diff
@@ -1,4 +1,6 @@
 ({
+    'guangfu.bjx.com.cn': { _name: '北极星光伏', '.': [{ title: '分类', docs: 'https://docs.rsshub.app/others.html#bei-ji-xing-guang-fu', source: ['/:type', '/'], target: '/bjxgf/:type?' }] },
```

Comment:
Do not add radar rules here.
You should add the rules in `lib/v2/bjxgf/radar.js`